apps/web/pages/event-types/[type].tsx

```diff
@@ -1836,6 +1852,18 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                   <LinkIcon className="h-4 w-4 text-neutral-500 ltr:mr-2 rtl:ml-2" />
                   {t("copy_link")}
                 </button>
+                {hashedLinkVisible && (
+                  <button
+                    onClick={() => {
+                      navigator.clipboard.writeText(placeholderHashedLink);
+                      showToast("Private link copied!", "success");
```

Comment:
Translation isn't used here. Also, The two messages are different. Here it says "Private link" and a little above it says "Link"

Reply:
Also, maybe it would be a good idea to allow copying the private link from event-types page. It would be a common action
<img width="1546" alt="Screenshot 2022-05-07 at 7 40 24 AM" src="https://user-images.githubusercontent.com/1780212/167233801-c5c4b6ff-e1ea-41ed-8268-ba767bf0254e.png">

Reply:
One more thing, maybe we should enable it by default. It would be good to have the private link ready. It would also make the UX simpler IMHO. Right now "Copy Private Link" CTA is not there till the time user actually goes to event-type and enable Private link. This small change would allow that CTA to be always there.

Also, in general, I find it confusing as a customer to understand the conditions why certain CTAs suddenly become invisible. We should make such CTA disabled and show a tooltip why it's disabled.

Reply:
Bug: https://www.loom.com/share/a111758664194c95872ce1ef5d623814

Copying is freely allowed from one CTA but not from the other one. In this case it shouldn't have been allowed as I didn't save

Reply:
Good catch. I think we do need consistency there.

Reply:
> One more thing, maybe we should enable it by default. It would be good to have the private link ready. It would also make the UX simpler IMHO. Right now "Copy Private Link" CTA is not there till the time user actually goes to event-type and enable Private link. This small change would allow that CTA to be always there.
> 
> Also, in general, I find it confusing as a customer to understand the conditions why certain CTAs suddenly become invisible. We should make such CTA disabled and show a tooltip why it's disabled.

Another thing best left to @Jaibles

Reply:
> Translation isn't used here. Also, The two messages are different. Here it says "Private link" and a little above it says "Link"

Yeah, good catch

Reply:
> Also, maybe it would be a good idea to allow copying the private link from event-types page. It would be a common action
> <img alt="Screenshot 2022-05-07 at 7 40 24 AM" width="1546" src="https://user-images.githubusercontent.com/1780212/167233801-c5c4b6ff-e1ea-41ed-8268-ba767bf0254e.png">

I think it might be useful but then again, maybe we’d prefer disposable links once they’re ready instead of private link.